src/applications/personalization/search-representative/config/search/searchRepresentativeWidget.jsx

```diff
@@ -1,49 +1,147 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { connect } from 'react-redux';
 import { setData } from 'platform/forms-system/src/js/actions';
 import { fetchRepresentativeSearchResults } from '../../actions';
+import SearchRepresentativeResult from './searchRepresentativeResult';
+import Pagination from '@department-of-veterans-affairs/component-library/Pagination';
+import { chunk } from 'lodash';
 
 const SearchRepresentativeWidget = props => {
   const { loading, representatives, formData } = props;
+  const [pages, setPages] = useState(1);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [maxPerPage] = useState(3);
```

Comment:
Is it better to use a normal const for this without `useState`?

Reply:
Yeah I think I just went overboard with doing it the "reacty" way LOL